components/homme/src/share/cxx/SphereOperators.hpp

```diff
@@ -1176,6 +1176,273 @@ class SphereOperators
   Real m_scale_factor_inv, m_laplacian_rigid_factor;
 };
 
+using TeamPolicy = Kokkos::TeamPolicy<ExecSpace>;
+using Team = TeamPolicy::member_type;
+
+static constexpr int NPNP = NP * NP;
+static constexpr int WARP_SIZE = warpSize;
```

Comment:
I think warpSize is a HIP specific variable?

Reply:
I think Cuda has it, but it's not a compile-time constant. I need a compile-time workaround. `#ifdef`? Other suggestions?

Reply:
It's also not defined for serial/openmp, which means we do need some sort of CPP guard. I guess `#ifdef` would do.